tsconfig.json

```diff
@@ -21,5 +21,13 @@
     "core/templates/dev/head/filters/string-utility-filters/convert-to-plain-text.filter.ts",
     "core/templates/dev/head/filters/string-utility-filters/truncate-at-first-line.filter.ts"
   ],
-  "include": ["extensions", "typings"]
+  "include": [
+    "extensions/ckeditor_plugins", 
+    "extensions/classifiers",
+    "extensions/interactions",
+    "extensions/objects",
+    "rich_text_components",
```

Comment:
This looks odd. Should it be extensions/rich_text_components?

Reply:
Yes, I will fix this after I merge with the RTCs PR.